README.md

```diff
@@ -21,26 +23,26 @@ tar -zxvf cf.tgz
 sudo mv cf /usr/local/bin
 ```
 
-Confirm the installation:
+### Confirm the installation
 
 ```bash
 cf --version
 ```
 
 As of this writing the current cf CLI version is `6.10.0-b78bf10`.
-	
+
 ## Setting up your account
 
-You will need a Cloud Foundry account before continuing. Please create an issue in the [DevOps issue tracker](https://github.com/18F/DevOps/issues) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
+You will need a Cloud Foundry account before continuing. Please [create an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
 
-To login, run:
+### Log in
 
 ```bash
 cf api --skip-ssl-validation https://api.cf.18f.us
```

Comment:
Seems so:

```
$ cf api https://api.cf.18f.us
Setting api endpoint to https://api.cf.18f.us...
FAILED
Invalid SSL Cert for api.cf.18f.us
TIP: Use 'cf api --skip-ssl-validation' to continue with an insecure API endpoint
```

See https://trello.com/c/eFV5ze6b/16-get-ssl-cert-for-api-labs-18f-us.